lib/class-wp-rest-menu-items-controller.php

```diff
@@ -22,6 +22,83 @@ public function __construct( $post_type ) {
 		$this->namespace = '__experimental';
 	}
 
+	/**
+	 * Overrides the route registration to support "allow_batch".
+	 *
+	 * @since 9.0.0
+	 */
+	public function register_routes() {
```

Comment:
I understand why this code had to be copied and pasted, still I would love to get rid of it (maybe not in this PR). A few quick ideas I have is updating `WP_REST_Posts_Controller::register_routes()` to accept an optional '$allow_batch' argument. We could also add a filter to `register_rest_route`.

Reply:
Yeah its really not great. My current thinking is we'll just make all posts controllers opt in to batching when we do the Core merge.